.gitignore

```
@@ -1,3 +1,6 @@
 /build
 /out
-/.vs
\ No newline at end of file
+/.vs
+.cache/
+compile_commands.json
+install_dir/
```

I think I know why you've added this and don't have much against it besides it being very specific to your workflow, but could we at least keep the `.gitignore` consistent to use semi-absolute paths instead of ignoring all files with these names?

```suggestion
/.cache
/compile_commands.json
/install_dir
```